src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -29,15 +29,32 @@
     </div>
     <div v-else>
       <v-card flat>
-        <v-radio-group v-model="agreementType" @change="changeAgreementType" class="agreement-option-list">
-          <v-radio v-for="(item, index) in getIncorporationAgreementDocuments"
-            :key="index" :value="item.code" :id="`agreement-type-${item.code}`"
+        <template v-if="isTypeCC">
+          <v-checkbox
+            v-for="(item, index) in getIncorporationAgreementDocuments"
+            :id="`agreement-type-${item.code}`"
+            class="agreement-option-list"
+            :key="index"
+            v-model="agreementType"
+            :value="item.code"
```

Comment:
Oh? It's possible to use both v-model and :value ?

Reply:
It is! In this case, as similar to the radio btns, let me look into this a bit tho!

Reply:
I wonder if this just happens to work, or if it's a good way to set the v-model initially?

Anyway, as long as this code works as expected, carry on (no change needed).

Reply:
Thats a good question! 
I'm a little suspicious of it as well, now that you mentioned it because i can't exactly differentiate between to the two values/purposes. 
It does work as expected, as currently the `value` is used locally to determine the the output text for the Summary/Review page and the model is being set to store. 🤷

Reply:
Ah i see, sorry the model is a boolean and the value is the string(code) used to determine the descriptions. 
Applies to both checkboxes and radio btns in this case.

Reply:
Oh, _that_ value. I confused it with the boolean.
Resolving this conversation then.